Python/ceval.c

```diff
@@ -1733,6 +1733,15 @@ _PyEval_EvalFrameDefault(PyFrameObject *f, int throwflag)
                 exc = POP(); /* exc */
                 /* fall through */
             case 0:
+#if defined(_M_ARM) && !defined(DEBUG) && (_MSC_VER < 1915)
```

Comment:
I have confirmed that the arm optimization bug is fixed on newer builds of visual studio. 
Does it make sense to check for older versions of the tools with a #if like this or should this block of code just go away?

Reply:
Let's just remove it.